docs/apache-airflow/faq.rst

```diff
@@ -41,13 +41,13 @@ There are very many reasons why your task might not be getting scheduled. Here a
   "airflow" and "DAG" in order to prevent the DagBag parsing from importing all python
   files collocated with user's DAGs.
 
-- Is your ``start_date`` set properly? The Airflow scheduler triggers the
-  task soon after the ``start_date + schedule_interval`` is passed.
+- Is your ``start_date`` set properly? The Airflow scheduler won't trigger the task until the
+  after the first schedule interval following the start date has passed.
```

Comment:
Might be worth adding a simple example here, since the "start_date+schedule_interval" doesn't work as cleanly any longer.

Reply:
reworded to clarify context